Classes/Hooks/Form.php

```diff
@@ -83,7 +83,7 @@ public function afterSubmit(FormRuntime $runtime, $element, $value, $requestArgu
             return '';
         }
 
-        if ((int)($expirationTime + intval($extensionSettings['crAdditionalExpiration'] ?? 3600)) <= time()) {
+        if ($GLOBALS['EXEC_TIME'] > ((int)$expirationTime + intval($extensionSettings['crAdditionalExpiration'] ?? 3600))) {
```

Comment:
`$GLOBALS['EXEC_TIME']` has been replaced with #25 and the int typecast is not required here anymore since 3ed47ef